e2e-tests/native/test/helpers/Gestures.ts

```diff
@@ -0,0 +1,179 @@
+// For reference https://github.com/webdriverio/appium-boilerplate/blob/main/tests/helpers/Gestures.ts
+import { RectReturn } from '@wdio/protocols/build/types'
+import { ChainablePromiseElement } from 'webdriverio'
+
+/**
+ * To make a Gesture methods more robust for multiple devices and also
+ * multiple screen sizes the advice is to work with percentages instead of
+ * actual coordinates. The percentages will calculate the position on the
+ * screen based on the SCREEN_SIZE which will be determined once if needed
+ * multiple times.
+ */
+
+let SCREEN_SIZE: RectReturn | null
```

Comment:
I think this should be a variable of Gesture class. Else you can not test on multiple devices with different screen sizes in parallel.

Reply:
I can move it into the Gesture class, but as all methods are static the `SCREEN_SIZE` would also need to be static. The reasoning behind that is that we don't want to recompute `SCREEN_SIZE`. However I am not sure if it would really not be possible to tes on multiple devices, as I think wdio starts a new process for each capability passed.